e2e/support/config.js

```diff
@@ -169,6 +173,14 @@ const defaultConfig = {
 
 const mainConfig = {
   ...defaultConfig,
+  ...(isEmbeddingSdk
+    ? {
+        chromeWebSecurity: true,
```

Comment:
We set this to `false` on all settings, and this disables `cross-site` checks, so CORS have always been ignored. We need to set this to `true` to actually test CORS.